src/Motor.Extensions.Hosting.Publisher/PublisherBuilder.cs

```diff
@@ -14,6 +15,8 @@ public class PublisherBuilder<TOutput> : IPublisherBuilder<TOutput>
     where TOutput : class
 {
     private readonly IServiceCollection _serviceCollection;
+    private IConfiguration? _publisherOptions;
```

Comment:
nit: why not calling this `_config` or `_configSection`? It is not directly related to the `IOptions<>` API?